components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/ApplicationsApiServiceImpl.java

```diff
@@ -871,6 +871,11 @@ public Response applicationsApplicationIdGenerateKeysPost(String applicationId,
                             jsonParamObj.put(APIConstants.JSON_ADDITIONAL_PROPERTIES, jsonContent);
                         }
                     }
+
+                    if (body.getCallbackUrl() != null && StringUtils.isNotEmpty(body.getCallbackUrl())) {
```

Comment:
StringUtils.isNotEmpty() has an inbuilt null check. Hence, null check is redundant.

Reply:
Resolved with https://github.com/wso2/carbon-apimgt/pull/13033/commits/aa09eada94f2cc723c9e6d6d6fa421318a84d3f0.